opensrp-reveal/src/main/java/org/smartregister/reveal/util/Utils.java

```diff
@@ -161,4 +150,63 @@ public static String getAge(String dob) {
     public static Boolean getDrawOperationalAreaBoundaryAndLabel() {
         return Boolean.valueOf(getGlobalConfig(CONFIGURATION.DRAW_OPERATIONAL_AREA_BOUNDARY_AND_LABEL, CONFIGURATION.DEFAULT_DRAW_OPERATIONAL_AREA_BOUNDARY_AND_LABEL.toString()));
     }
+
+    /**
+     * Creates a circle using a GeoJSON polygon.
+     * It's not strictly a circle but by increasing the number of sides on the polygon you can get pretty close to one.
+     *
+     * Adapted from https://stackoverflow.com/questions/37599561/drawing-a-circle-with-the-radius-in-miles-meters-with-mapbox-gl-js/39006388#39006388
+     *
+     * @param center - Coordinates for the center of the circle
+     * @param radius - Radius of the circle in meters
+     * @param points - Since this is a GeoJSON polygon, we need to have a large number of sides
+     *               so that it gets as close as possible to being a circle
+     * @return
+     * @throws Exception
+     */
+
+    public static FeatureCollection createGeoJSONCircle(LatLng center, Float radius, Float points) throws JSONException {
```

Comment:
This is returning only 1 feature, so should return a feature and not feature Collection.
Also should be renamed to `createCircleFeature` since its returns a Feature